IPython/html/static/base/less/page.less

```diff
@@ -127,3 +132,9 @@ span#login_widget > .button,
     .fa();
     content: @ico;
 }
+
+@media (min-width: @screen-sm-min){
+    .container {
```

Comment:
This does change the maximum width of the notebook to 768px, when it used to be 1140px, so it's a dramatic change. Was that intentional? Or did you mean `min-width: @screen-sm-min` and `max-width: @screen-sm-max`?